osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -13,17 +16,46 @@ public class PauseOverlay : GameplayMenuOverlay
     {
         public Action OnResume;
 
+        public override bool IsPresent => base.IsPresent || pauseLoop.IsPlaying;
```

Comment:
I think this is causing input to be handled for longer and thus breaking tests.

Reply:
I've ran the failing test a few times locally and it does fail sometimes, but not always. The autoplay tests which are failing seem to be very sensitive to update framerate (try running them in browser with window focus/no window focus - for me at least the first option is lengthy but mostly fits within the allotted timeframe while the other can fail fairly often on one of the seeks).